docs/source/_toctree.yml

```diff
@@ -9,24 +9,26 @@
 
 - title: Task guides
   sections:
-  - local: task_guides/image_classification_lora
-    title: Image classification using LoRA
   - local: task_guides/seq2seq-prefix-tuning
     title: Prefix tuning for conditional generation
   - local: task_guides/clm-prompt-tuning
     title: Prompt tuning for causal language modeling
-  - local: task_guides/semantic_segmentation_lora
-    title: Semantic segmentation using LoRA
   - local: task_guides/ptuning-seq-classification
     title: P-tuning for sequence classification
-  - local: task_guides/dreambooth_lora
-    title: Dreambooth fine-tuning with LoRA
-  - local: task_guides/token-classification-lora
-    title: LoRA for token classification
-  - local: task_guides/int8-asr
-    title: int8 training for automatic speech recognition
-  - local: task_guides/semantic-similarity-lora
-    title: Semantic similarity with LoRA
+  - title: LoRA
```

Comment:
I like this structure, nice.

Strangely, in the doc preview, this section shows as all capitalized "LORA" for me, not sure why. Not a big deal, just curious.